### PR TITLE
Support Night Mode

### PR DESCRIPTION
**Pull request by andrewpeters9**

### CODE REVIEW

File: src/__init__.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def initCSS(self, *args):
       transform: translateY(-0.2em);
       font-size: .6em;
       -webkit-text-stroke-width: initial;
-      color: #666;
+      color: inherit;
```
Comment (andrewpeters9) [inline src/__init__.py]: Alternative would be to use this `.nightMode` class: https://docs.ankiweb.net/templates/styling.html#night-mode

**Review by andrewpeters9 — COMMENTED** [review]
Tested via adding CSS manually to the cards

**Review by phu54321 — APPROVED** [review]

**Comment by phu54321** [comment]
I hope we could just set

```css
    color: var(--fg);
```

but idk how compatible this would be on lower end anki version...

**Comment by andrewpeters9** [comment]
> ```css
> color: var(--fg);
> ```

When I test that I get this:

image

**Comment by andrewpeters9** [comment]
@phu54321 Could we merge and refactor later?

**Pull request merged**